config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  resources :guests, only:[:show, :update]
+  resources :rooms, only: [:show]
```

Comment:
Great job attempting to implement the `resources` syntax, there are only a few that aren't implemented according to the user stories.

User story 2 is for the Rooms index page, in your `routes.rb` you have a show route.

For user story 3, this is an association between a room and a guest, so you wither want a nested association route:

```ruby
resources :guests, only: :show do
    resources :rooms, only: :create
end

resources :rooms, only: :index
```

or 

```ruby
resources :guests, only: :show
resources :rooms, only: index
resources :guest_rooms, only: :create
```

The biggest thing I want to emphasize is that we are adding using a `POST` to `create` because we want to create an association between a Guest and a Room.